### PR TITLE
Shortcut for focusing omnibox and changing tabs

### LINKED ISSUE #1498: [classname selector] Topmenu shortcut should focus select dropdown

### PR DESCRIPTION
**Pull request by Rheeseyb**
Fixes #1498 

**Problem:**
We want a quick way to focus the omnibox and toggle the selected tab

**Fix:**
Map the shortcut `cmd`+`/` to the action for focusing the topmenu, and also capture that shortcut when the topmenu is focused to trigger an action for changing the selected tab.

omnibox-shortcut

### CODE REVIEW

**Review by balazsbajorics — APPROVED** [review]
please leave some TODO comments around `useFocusOnCountIncrease` because we definitely shouldn't need to use a setTimeout(0) for `EditorActions.focusFormulaBar` to work properly (it's probably my bad and I have some ideas why it might not work)

**Review by enidemi — APPROVED** [review]

**Comment by enidemi** [comment]
sometimes selecting a scene on the canvas opens the dropdown automatically, is it remounting?

**Comment by Rheeseyb** [comment]
> sometimes selecting a scene on the canvas opens the dropdown automatically, is it remounting?

Ah crap it must be. Hmmm I guess that means a different approach is needed in the classname-select

File: editor/src/components/canvas/controls/formula-bar.tsx [hunk 1]
```diff
@@ -106,6 +111,26 @@ export const FormulaBar = betterReactMemo('FormulaBar', () => {
...
+  const editorStoreRef = useRefEditorState((store) => store)
+  const onKeyDown = React.useCallback(
+    (event: React.KeyboardEvent<HTMLElement>) => {
+      const namesByKey = applyShortcutConfigurationToDefaults(
+        editorStoreRef.current.userState.shortcutConfig,
+      )
+      handleShortcuts(namesByKey, event.nativeEvent, null, {
+        [TOGGLE_FOCUSED_OMNIBOX_TAB]: () => {
+          event.stopPropagation()
+          event.preventDefault()
```
Comment (seanparsons) [inline editor/src/components/canvas/controls/formula-bar.tsx:123]: I don't _think_ this is necessary anymore, because it's in `handleShortcuts`.

**Review by seanparsons — APPROVED** [review]

**Review by balazsbajorics — APPROVED** [review]

File: editor/src/components/canvas/controls/formula-bar.tsx (hunk 1 above)
Comment (Rheeseyb) [reply editor/src/components/canvas/controls/formula-bar.tsx:123]: Ah good catch. It is indeed unnecessary, thanks!

**Pull request merged**